NickvisionTubeConverter.Shared/Models/Download.cs

```diff
@@ -221,9 +223,11 @@ public void Start(bool useAria, bool embedMetadata, string? cookiesPath, Localiz
                     dynamic ariaDict = new PyDict();
                     dynamic ariaParams = new PyList();
                     ariaParams.Append(new PyString($"--max-overall-download-limit={(_limitSpeed ? _speedLimit : 0)}K"));
-                    ariaParams.Append(new PyString("--allow-overwrite=true"));
+                    ariaParams.Append(new PyString($"--allow-overwrite={_overwriteFiles}"));
```

Comment:
For aria `allow-overwrite` should always be true to be able to restart downloads.
Also I will improve UI a bit.